go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -513,6 +513,37 @@ func (s *VtctldServer) backupTablet(ctx context.Context, tablet *topodatapb.Tabl
 	}
 }
 
+// CancelSchemaMigration is part of the vtctlservicepb.VtctldServer interface.
+func (s *VtctldServer) CancelSchemaMigration(ctx context.Context, req *vtctldatapb.CancelSchemaMigrationRequest) (resp *vtctldatapb.CancelSchemaMigrationResponse, err error) {
+	span, ctx := trace.NewSpan(ctx, "VtctldServer.CancelSchemaMigration")
+	defer span.Finish()
+
+	defer panicHandler(&err)
+
+	span.Annotate("keyspace", req.Keyspace)
+	span.Annotate("uuid", req.Uuid)
+
+	query, err := alterSchemaMigrationQuery("Cancel", req.Uuid)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Info("Calling ApplySchema to Cancel migration")
```

Comment:
i don't have an opinion _which_ way we go, but i wrote the log lines for retry/cleanup as lowercase (e.g. "Calling ApplySchema to retry migration"). we should either uppercase mine, or lowercase these for consistency, I defer to your preference on which!

Reply:
Lower case it is.